assets/img/pipeline-comparison.svg

```diff
@@ -0,0 +1 @@
,40.67901234567901,0)" xlink:href="#y"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,53.02469135802469,0)" xlink:href="#C"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,59.19753086419753,0)" xlink:href="#t"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,72.71604938271605,0)" xlink:href="#ac"/></g><pattern id="m" patternUnits="userSpaceOnUse" x="1142" y="806.84" width="46.33" height="52.64"><use xlink:href="#ad" transform="translate(0, 0) scale(46.33333333333326,52.64048865619538)"/></pattern><path fill="#333" d="M121-226c-27-7-43 5-38 36h38v33H83V0H34v-157H6v-33h28c-9-59 32-81 87-68v32" id="ae"/><path fill="#333" d="M220-157c-53 9-28 100-34 157h-49v-107c1-27-5-49-29-50C55-147 81-57 75 0H25l-1-190h47c2 12-1 28 3 38 10-53 101-56 108 0 13-22 24-43 59-42 82 1 51 116 57 194h-49v-107c-1-25-5-48-29-50" id="af"/><g id="n"><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,0,0)" xlink:href="#H"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,13.037037037037036,0)" xlink:href="#y"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,26.617283950617285,0)" xlink:href="#z"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,36.123456790123456,0)" xlink:href="#z"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,45.629629629629626,0)" xlink:href="#w"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,59.20987654320987,0)" xlink:href="#ae"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,67.29012345679013,0)" xlink:href="#t"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,82.1604938271605,0)" xlink:href="#z"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,91.66666666666667,0)" xlink:href="#af"/></g><pattern id="o" patternUnits="userSpaceOnUse" x="690" y="806.84" width="46.33" height="52.64"><use xlink:href="#ad" transform="translate(0, 0) scale(46.33333333333326,52.64048865619538)"/></pattern><path fill="#333" d="M175 0L67-191c6 58 2 128 3 191H24v-248h59L193-55c-6-58-2-129-3-193h46V0h-61" id="ag"/><g id="p"><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,0,0)" xlink:href="#ag"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,17.586419753086417,0)" xlink:href="#x"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,24.37654320987654,0)" xlink:href="#v"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,32.456790123456784,0)" xlink:href="#z"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,41.962962962962955,0)" xlink:href="#x"/><use transform="matrix(0.06790123456790123,0,0,0.06790123456790123,48.753086419753075,0)" xlink:href="#G"/></g><pattern id="q" patternUnits="userSpaceOnUse" x="903.17" y="926.79" width="46.33" height="52.75"><use xlink:href="#ah" transform="translate(0, 0) scale(46.33333333333326,52.74871794871786)"/></pattern></defs></g></svg>
```

Comment:
Including the runtime component of nitric (membrane) to this is a nice touch.

I think unifying and simplifying the pipeline would be good, adding the container build and local run as separate branching steps seems like extra detail that might not be needed for the comparison, and local testing `nitric run` is part of the development step (at least for me conceptually).

I think something like this could be simpler and more focussed on high level lifecycle:
```
|-----------nitric-------------------|
             |--terrafrom--|
Development -> Deployment -> Runtime
```

Reply:
Have updated existing digram. Lucid had a template, but would take to long to complete. Can replace with cycle diagram when we get a designer to review diagrams.